src/api/analytics.ts

```diff
@@ -49,9 +49,35 @@ export type AnalyticsOptions = {
     enrollmentDate?: string;
 };
 
+type KnownAscDescValues =
+    | "ouname"
+    | "programstatus"
+    | "createdbydisplayname"
+    | "lastupdatedbydisplayname"
+    | "enrollmentdate"
+    | "incidentdate"
+    | "lastupdated";
+
+type AscDescParameter = KnownAscDescValues | Id;
+
+type PageOptions = {
+    totalPages: boolean;
+    page: number;
+    pageSize: number;
+    paging: boolean;
+};
```

Comment:
We can now use pagination options in the request, and I see that the response has a `.metaData.pager` object. I guess we don't need to access that object at the moment for simplicity, right?